cmd/fluxd/main.go

```diff
@@ -106,6 +106,10 @@ func main() {
 		registryTrace        = fs.Bool("registry-trace", false, "output trace of image registry requests to log")
 		registryInsecure     = fs.StringSlice("registry-insecure-host", []string{}, "use HTTP for this image registry domain (e.g., registry.cluster.local), instead of HTTPS")
 
+		// AWS authentication
```

Comment:
LGTM. Plural vs singular is alway the question, and looks like fluxd is sticking to singular, e.g. `--registry-insecure-host` above.

Reply:
I was think perhaps I ought to make these more specific -- taken in isolation, the flag names suggest they might pertain to other AWS uses. Maybe:

 - `--registry-ecr-region`
 - `--registry-ecr-account-id`

(those are getting pretty long-winded, though the 80% scenario is you don't have to supply them)

Reply:
After adding a flag for _excluding_ registries, so the EKS system registry can be excluded, I changed this to `--registry-ecr-include-id`, with the exclude flag being `--registry-ecr-exclude-id`.